utils/build-toolchain-tensorflow

```diff
@@ -82,12 +86,20 @@ while [ $# -ne 0 ]; do
     usage
     exit 0
   ;;
+  -r|--release)
+    shift
+    S4TF_RELEASE_VERSION=$1
+    if [ -z "$S4TF_RELEASE_VERSION" ]; then
```

Comment:
Why POSIX rather than bash? (i.e. `[` vs `[[`)

Reply:
No actual reason - I copied `[` from elsewhere in the file, hope that's okay.

The file itself isn't consistent in its usage of `[` vs `[[`.